packages/run-it/src/components/ConfigForm/ConfigForm.spec.tsx

```diff
@@ -28,27 +28,24 @@ import React from 'react'
 import { fireEvent, screen, waitFor } from '@testing-library/react'
 import { renderWithTheme } from '@looker/components-test-utils'
 import userEvent from '@testing-library/user-event'
-import { defaultConfigurator } from '..'
-import { runItNoSet, initRunItSdk } from '../..'
+import { BrowserAdaptor, registerTestEnvAdaptor } from '@looker/extension-utils'
+
+import { initRunItSdk, runItNoSet } from '../..'
 import { ConfigForm, loadSpecsFromVersions, RunItConfigKey } from '.'
 
 describe('ConfigForm', () => {
-  const sdk = initRunItSdk()
+  const adaptor = new BrowserAdaptor(initRunItSdk())
+  registerTestEnvAdaptor(adaptor)
+
   const apiLabel = /API server URL/i
   const authLabel = /OAuth server URL/i
-
   beforeEach(() => {
-    defaultConfigurator.removeStorage(RunItConfigKey)
+    localStorage.removeItem(RunItConfigKey)
```

Comment:
Should this be using an adaptor storage call or do we presume the unit tests will always be browser based?

Reply:
This PR modifies `ConfigForm` to use `localStorage` directly, not via the envAdaptor. The reason being that localstorage interactions would otherwise be async, requiring a significant amount of changes. Once we have redux in place this will be delegated to sagas, making it much easier. Also `ConfigForm` is entirely for browser apps right now as extensions do not need it.